lib/blocktree/mocks_generate_test.go

```diff
@@ -0,0 +1,6 @@
+// Copyright 2022 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package blocktree
+
+//go:generate mockgen -destination=mock_instance_test.go -package $GOPACKAGE github.com/ChainSafe/gossamer/lib/runtime Instance
```

Comment:
generate to mocks_test.go for consistency with other packages (also don't forget to re-`go generate`)

```suggestion
//go:generate mockgen -destination=mocks_test.go -package $GOPACKAGE github.com/ChainSafe/gossamer/lib/runtime Instance
```